src/plugin/cesium/cesiumrenderer.js

```diff
@@ -426,6 +426,12 @@ plugin.cesium.CesiumRenderer.prototype.renderSync = function() {
   }
 };
 
+/**
+ * @inheritDoc
+ */
+plugin.cesium.CesiumRenderer.prototype.supportsVideoOverlay = function() {
+  return false;
+};
```

Comment:
This might be better if moved to abstractwebglrenderer.js. That would cover any class that extends it as a base implementer of the interface.